ivy_tests/test_ivy/test_functional/test_extensions/test_core/test_creation.py

```diff
@@ -36,3 +37,36 @@ def test_triu_indices(
         k=k,
         device=device,
     )
+
+
+@handle_cmd_line_args
```

Comment:
We should be using the new decorator `handle_test`, look here for more info: https://lets-unify.ai/ivy/deep_dive/ivy_tests.html#ivy-test-decorators

Reply:
Done.